public/app/features/playlist/PlaylistPage.tsx

```diff
@@ -87,13 +84,6 @@ export const PlaylistPage: FC<PlaylistPageProps> = ({ navModel }) => {
   return (
     <Page navModel={navModel}>
       <Page.Contents isLoading={loading}>
-        {hasPlaylists && (
-          <PageActionBar
-            searchQuery={searchQuery}
-            linkButton={{ title: 'New playlist', href: '/playlists/new' }}
-            setSearchQuery={setSearchQuery}
-          />
-        )}
```

Comment:
with this you also removed the button to create new playlists :)

Reply:
ah...I see.
Great catch!